lib/app/utils/hl7_validator.rb

```diff
@@ -3,6 +3,11 @@
 module Inferno
   # A validator that calls out to the HL7 validator API
   class HL7Validator
+    DETAILS_FILTER = [
```

Comment:
nit: I think `ISSUES_FILTER` better expresses the intention and is consistent with the naming used through the rest of the file (e.g. `issues_by_severity`)
```suggestion
    ISSUES_FILTER = [
```

Reply:
My concern is that this is for filtering detail text. It is used at condition expression `|filter| filter.match?(iss.details.text)`
Filters other issue elements, such as expression, or code, have different filter expression.

Reply:
I changed the name to `ISSUE_DETAILS_FILTER`